src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayPeopleInControlController.cs

```diff
@@ -39,15 +39,50 @@ public async Task<IActionResult> GetGatewayPeopleInControlPage(Guid applicationI
             [HttpPost("/Roatp/Gateway/{applicationId}/Page/PeopleInControl")]
             public async Task<IActionResult> EvaluatePeopleInControlPage(PeopleInControlPageViewModel viewModel)
             {
+                var validationCheck = await GatewayValidator.Validate(viewModel);
+
+                if (validationCheck.Errors == null || !validationCheck.Errors.Any())
+                    return await SubmitGatewayPageAnswer(viewModel, $"{GatewayViewsLocation}/PeopleInControl.cshtml",
+                        validationCheck.Errors);
+
                 var username = _contextAccessor.HttpContext.User.UserDisplayName();
-                var vmRebuild = await _orchestrator.GetPeopleInControlViewModel(new GetPeopleInControlRequest(viewModel.ApplicationId, username));
+                var vmRebuild =
+                    await _orchestrator.GetPeopleInControlViewModel(
+                        new GetPeopleInControlRequest(viewModel.ApplicationId, username));
                 viewModel.CompanyDirectorsData = vmRebuild?.CompanyDirectorsData;
                 viewModel.PscData = vmRebuild?.PscData;
                 viewModel.TrusteeData = vmRebuild?.TrusteeData;
                 viewModel.WhosInControlData = vmRebuild?.WhosInControlData;
```

Comment:
I'm a bit confused why this bit is here...won't the reloaded viewmodel properties just be discarded?  Submit will pass as the validation has already been done so why it the vmRebuild needed?

Reply:
No the rebuild is being done because it failed validiation, and for this page these parts are not primitive types so can't be embedded in a hidden, so needs rebuilding.

Reply:
Yeah, it's because I misread the above bit.  What'd probably make more sense in my head would be to do the rebuild of the VM if validation failed and then call SubmitGatewayPageAnswer once at the end regardless of whether validation passed or failed and let it do the redirection.